include/rmm/mr/device/cuda_async_memory_resource.hpp

```diff
@@ -60,7 +60,21 @@ class cuda_async_memory_resource final : public device_memory_resource {
     posix_file_descriptor = 0x1,  ///< Allows a file descriptor to be used for exporting. Permitted
                                   ///< only on POSIX systems.
     win32     = 0x2,              ///< Allows a Win32 NT handle to be used for exporting. (HANDLE)
-    win32_kmt = 0x4  ///< Allows a Win32 KMT handle to be used for exporting. (D3DKMT_HANDLE)
+    win32_kmt = 0x4,  ///< Allows a Win32 KMT handle to be used for exporting. (D3DKMT_HANDLE)
+    fabric    = 0x8   ///< Allows a fabric handle to be used for exporting. (cudaMemFabricHandle_t)
+  };
+
+  /**
+   * @brief Flags for specifying the memory pool accessibility from other devices.
+   *
+   * @note The default, `prot_none`, marks the pool's memory as private to the device in
+   * which it was created. `prod_read` and `prod_read_write` should only be use if memory
+   * sharing among devices is required.
+   */
+  enum class access_flags {
+    prot_none       = 0x0,  ///< Default, make pool not accessible. (cudaMemAccessFlagsProtNone)
+    prot_read       = 0x1,  ///< Make pool read accessible. (cudaMemAccessFlagsProtRead)
+    prot_read_write = 0x2   ///< Make pool read-write accessible. (cudaMemAccessFlagsProtReadWrite)
```

Comment:
https://docs.nvidia.com/cuda/cuda-runtime-api/group__CUDART__TYPES.html says `cudaMemAccessFlagsProtReadWrite = 3`.
```suggestion
    prot_read_write = 0x3   ///< Make pool read-write accessible. (cudaMemAccessFlagsProtReadWrite)
```

Reply:
Already fixed in 1077dc5. :)

Reply:
yes, but nice catch nonetheless.